### ISSUE TITLE
Field for slowdown% (LLS/MS)

### ISSUE THREAD

**Issue by KirbyComment** [n-rook/thscoreboard#356]
LLS and MS have slowdown percentages tracked but they don't support replays. There's however no way to manually enter the slowdown percentage when submitting a score.

**Comment by n-rook** [comment]
Yeah, I guess this makes sense to add, since some people (and old replays) have machines that struggle to keep up with PC-98 emulators. I hate asking users for stuff like this, but I think this one is probably worth it, although I'm still inclined to make it optional.
